### ISSUE TITLE
Date in OpenData_Slovakia_Covid_Deaths_AgeGroup_District.csv

### ISSUE THREAD

**Issue by jodik** [Institut-Zdravotnych-Analyz/covid19-data#92]
Hi all,

Thanks for making this data open! Wondering what does date mean in OpenData_Slovakia_Covid_Deaths_AgeGroup_District.csv as it seems to be somewhere between actual date of death and reported date of death (after medical evaluation).

See example below:
September:
- Actual Deaths? (Covid_Deaths_AgeGroup_District.csv): 28
- Extra deaths above expectations: 199 (https://www.startitup.sk/wp-content/uploads/2021/03/zomreti-podla-mesiacov-1024x507.jpg)

October:
- Reported Deaths (DeathsCumulative.csv - DeathCovid): 289
- Actual Deaths? (Covid_Deaths_AgeGroup_District.csv): **372**
- Extra deaths above expectations: **996** 

November:
- Reported Deaths (DeathsCumulative.csv): 695
- Actual Deaths? (Covid_Deaths_AgeGroup_District.csv): 1022
- Extra deaths above expectations: 1751

December:
- Reported Deaths (DeathsCumulative.csv): 1387
- Actual Deaths? (Covid_Deaths_AgeGroup_District.csv): 1756
- Extra deaths above expectations: 2766

If Covid_Deaths_AgeGroup_District.csv should reflect the actual date of death, then, the ratio between extra deaths and actual covid deaths should be somewhat stable. Does that make sense?

**Comment by matejmisik** [comment]
Hi there, the data on deaths is provided by the coroners office in 2 views as you correctly point out:

1. DeathsCumulative - this is the daily increment of newly processed deaths irrespective of the date of death
2. Covid_Deaths_AgeGroup_District - this is with the actual date of death, where often it can take weeks (especially in the peek) for the death to appear in this file. It includes only deaths the coroners office attributed to covid (with primary Dg covid) - either via autopsy or via medical condition / symptoms / test. It thus "misses" deaths that where not attributed to covid in one of these ways. Additionally the coroners office also has deaths "with" covid, which you can find as a column in the Cumulative report.

To summarize - yes, it is visible, that not nearly all deaths related to covid (directly or indirectly) where identified as such. This can be due the lack of healthcare, unwillingness to go to hospital for different conditions and several further factors.
